code/src/index.js

```diff
@@ -1,6 +1,513 @@
-import React from 'react'
+import React, { useState } from 'react'
 import ReactDOM from 'react-dom'
 import './index.css'
-import { App } from './App'
 
-ReactDOM.render(<App />, document.getElementById('root'))
+
+const thing = [
+  "broken code",
+  "phone number",
+  "coupons"
+]
+
+
+const thingsTwos = [
+  "toilet paper",
+  "eggs",
+  "yarn"
+]
+
+
+
+
+
+const App = () => {
+  const [holiday, setHoliday] = useState("");
+  const [nameOne, setNameOne] = useState("");
+  const [costumeOne, setCostumeOne] = useState("");
+  const [costumeTwo, setCostumeTwo] = useState("");
+  const [food, setFood] = useState("");
+  const [hustle, setHustle] = useState("");
+  const [things, setThings] = useState("");
+  const [thingsTwo, setThingsTwo] = useState()
+  const [phrase, setPhrase] = useState("")
+  const [summary, setSummary] = useState(false)
+
+  const handleSubmit = event => {
+    event.preventDefault();
+    setSummary(true);
+  };
+  return (
+
+
+    <div className="container">
+      <div className="header"><h1>Time For Halloween<br>
+      </br> Mad Lib</h1></div>
```

Comment:
We shouldn't use `br` tags at all, you can achieve same result by proper CSS styles (display property perhaps?)